lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -183,11 +187,31 @@ def wait_until_deployed
         if errorEvents > 0 then error("Deployment failed.") end
       end
 
+      def abort_update()
+        options = {
+          :environment_name  => env_name
+        }
+
+        begin
+          eb.abort_environment_update(options)
+        rescue; end
+
+        options[:attribute_names] = ["Status"]
+        while eb.describe_environment_health(options).status != "Ready"
+          sleep 5
```

Comment:
Is it possible that this call never returns `Ready`, and this method never returns?